index.html

```diff
@@ -1,22 +1,105 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <title>Reign</title>
     <script src="script.js"></script>
     <link rel="stylesheet" type="text/css" href="style.css">
-
   </head>
-  <body>
-      <div style="border: 1px solid black">Hello World</div>
-    <h1  class="text-color">Test</h1>
-    <button class="font">Click me</button>
-    <ol class="list">
-      <li>Luni</li>
-      <li>Marti</li>
-      <li>Miercuri</li>
-      <li>Joi</li>
-      <li>Vineri</li>
-      <li>Sambata</li>
-      <li>Duminica</li>
-    </ol>
+  <body background="bg-image.jpg">
+      <h1 class="intro">Welcome! I am a Reign fan, exactly like you!</h1>
```

Comment:
you must specify the path like that
`Images/bg-image.jpg` 

your  index.html file is at the same level as Image, but image you're trying to acces is inside of `Images` folder

Reply:
It works. Thank you very much.